### PR TITLE
Update governance contract with two token voting

### PR DESCRIPTION
**Pull request by yubo-ruan**

### CODE REVIEW

File: contracts/governance/GovernorAlpha.sol [hunk 1]
```diff
@@ -45,6 +45,9 @@ contract GovernorAlpha is ClaimableContract {
     // OLD: CompInterface public comp;
     TrustTokenInterface public trustToken;
```
[the remark this answers is not in the document]
Comment (hal909) [reply contracts/governance/GovernorAlpha.sol:48]: Also make VoteToken IVoteToken
contract VoteToken is IVoteToken { ... }

File: contracts/governance/GovernorAlpha.sol [hunk 2]
```diff
@@ -149,11 +152,12 @@ contract GovernorAlpha is ClaimableContract {
...
      * @dev Initialize sets the addresses of timelock contract, trusttoken contract, and guardian
      */
-    function initialize(address timelock_, address trustToken_, address guardian_, uint _votingPeriod) external {
+    function initialize(address timelock_, address trustToken_, address guardian_, address stkTRU_, uint256 votingPeriod_) external {
         timelock = TimelockInterface(timelock_);
         trustToken = TrustTokenInterface(trustToken_);
+        stkTRU = TrustTokenInterface(stkTRU_);
         guardian = guardian_;
-        votingPeriod = _votingPeriod;
+        votingPeriod = votingPeriod_;
```
Comment (hal909) [inline contracts/governance/GovernorAlpha.sol:162]: Change underscores to beginning of variable when passing into function

File: contracts/governance/VoteToken.sol [hunk 3]
```diff
@@ -12,9 +12,10 @@
 // OLD: pragma solidity ^0.5.16;
 pragma solidity 0.6.10;
 
-import {TimeLockedToken} from "../trusttoken/TimeLockedToken.sol";
+import {ClaimableContract} from "../trusttoken/common/ClaimableContract.sol";
+import {ERC20} from "../trusttoken/common/ERC20.sol";
 
-abstract contract VoteToken is TimeLockedToken {
+abstract contract VoteToken is ERC20, ClaimableContract {
```
Comment (hal909) [inline contracts/governance/VoteToken.sol:18]: Make VoteToken IVoteToken

File: contracts/governance/GovernorAlpha.sol [hunk 4]
```diff
@@ -149,10 +154,11 @@ contract GovernorAlpha is ClaimableContract {
     /**
      * @dev Initialize sets the addresses of timelock contract, trusttoken contract, and guardian
      */
-    function initialize(address timelock_, address trustToken_, address guardian_, uint _votingPeriod) external {
-        timelock = TimelockInterface(timelock_);
-        trustToken = TrustTokenInterface(trustToken_);
-        guardian = guardian_;
+    function initialize(address _timelock, address _trustToken, address _guardian, address _stkTRU, uint256 _votingPeriod) external {
```
Comment (vanruch) [inline contracts/governance/GovernorAlpha.sol:157]:
```suggestion
    function initialize(ITimelock _timelock, IVoteToken _trustToken, address _guardian, IVoteToken _stkTRU, uint256 _votingPeriod) external {
```

Comment (vanruch) [reply contracts/governance/GovernorAlpha.sol:157]: And remove casting in initializer code

**Review by vanruch — APPROVED** [review]

**Pull request merged**